cranelift/wasm/src/code_translator.rs

```diff
@@ -53,14 +53,14 @@ use wasmparser::{MemoryImmediate, Operator};
 /// Translates wasm operators into Cranelift IR instructions. Returns `true` if it inserted
 /// a return.
 pub fn translate_operator<FE: FuncEnvironment + ?Sized>(
-    module_translation_state: &ModuleTranslationState,
+    validator: &mut FuncValidator<impl WasmModuleResources>,
```

Comment:
One early comment (before i try this tomorrow): can you please expose `wasmparser::FuncValidator` publicly, so that embedders aren't required to add `wasmparser` to their own dependencies?

Reply:
Sure thing! I've pushed up a comment which reexports `wasmparser` as a crate since there's a number of types/etc you'll likely want to pull from there.